components/button/src/index.js

```diff
@@ -2,69 +2,142 @@ import styled from 'react-emotion';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withWhiteSpace } from '@govuk-react/hoc';
-import { NTA_LIGHT, SPACING } from '@govuk-react/constants';
+import {
+  BORDER_WIDTH_FORM_ELEMENT,
+  FOCUSABLE,
+  MEDIA_QUERIES,
+  SPACING_POINTS,
+} from '@govuk-react/constants';
+import { typography } from '@govuk-react/lib';
 import {
   BUTTON_COLOUR,
-  BUTTON_COLOUR_DARKEN_15,
   WHITE,
-  YELLOW,
 } from 'govuk-colours';
+import { darken, stripUnit } from 'polished';
 
-const BUTTON_COLOUR_DARKEN_5 = '#00692f'; // darken(#00823b, 5%)
+const BUTTON_SHADOW_SIZE = BORDER_WIDTH_FORM_ELEMENT;
+const RAW_SPACING_2 = stripUnit(SPACING_POINTS[2]);
+const RAW_BORDER_WIDTH = stripUnit(BORDER_WIDTH_FORM_ELEMENT);
+const RAW_SHADOW = stripUnit(BUTTON_SHADOW_SIZE);
+const HALF_SHADOW = RAW_SHADOW / 2;
+const BASE_PAD = RAW_SPACING_2 - RAW_BORDER_WIDTH;
 
 const StyledButton = styled('button')(
-  {
-    backgroundColor: BUTTON_COLOUR,
-    border: 'none',
-    boxShadow: `0 2px 0 ${BUTTON_COLOUR_DARKEN_15}`,
-    color: WHITE,
+  ({ isStart }) => typography.font({
+    size: isStart ? 24 : 19,
+    lineHeight: isStart ? '1' : '19px',
+    weight: isStart ? 'bold' : undefined,
+  }),
+  FOCUSABLE,
+
+  ({
+    buttonColour = BUTTON_COLOUR,
+    buttonHoverColour = darken(0.05, buttonColour),
```

Comment:
This is where button colours get calculated, if a prop isn't provided to override.

Reply:
ah yeah this is the default value, was reading it as an assignment